Src/Tests/Temporal.Sdk.Common.Tests/Common/TestPayload.cs

```diff
@@ -0,0 +1,68 @@
+using System;
```

Comment:
I didn't follow the structure under SDK because I didn't think my tests would be 1 to 1 with the classes. I expected more composition, so I laid them out according to functional area of the public API. I have no problem with changing the structure.
I prefixed everything with test because it makes navigation easier with vscode and Rider/Resharper. If I search for Payload, I'm guaranteed to get multiple results. If I search for TestPayload, I'm only going to get one. I went with the verbose names because I was hoping I could make them clear enough that seeing the test name and the failure would indicate what went wrong. However, including fluent assertions mitigates that, and I didn't consider the view from the unit test UI.

Reply:
I use VS, together we can make it convenient for both. I am generally fine with any structure, as long as it is self-explanotary and consistent. :) Let's discuss the details on Monday.

Reply:
Names discussed and adjusted.